subsys/net/lib/lwm2m/lwm2m_obj_binaryappdata.c

```diff
@@ -119,6 +122,11 @@ static int lwm2m_binaryappdata_init(void)
 	lwm2m_binaryappdata.create_cb = lwm2m_binaryappdata_create;
 	lwm2m_register_obj(&lwm2m_binaryappdata);
 
+	struct lwm2m_engine_obj_inst *inst;
```

Comment:
The commit message makes no mention of changes in this file - were they intended?

Reply:
Sorry, this was a test code that I accidentally added into the commit.
I needed a binary resource, large enough to cause block-wise.